composer.json

```
@@ -7,11 +7,15 @@
         "psr-4": {
           "EzSystems\\EzPlatformAdminUiBundle\\": "src/bundle/",
           "EzSystems\\EzPlatformAdminUi\\": "src/lib/",
+          "Ibexa\\Bundle\\AdminUi\\": "src/bundle/",
           "Ibexa\\AdminUi\\": "src/lib/"
         }
     },
     "autoload-dev": {
-        "psr-4": { "EzSystems\\EzPlatformAdminUi\\Tests\\": "src/lib/Tests" }
+        "psr-4": {
+            "EzSystems\\EzPlatformAdminUi\\Tests\\": "src/lib/Tests",
+            "Ibexa\\AdminUi\\Tests\\": "src/lib/Tests"
```

This does not follow [package structure](https://doc.ibexa.co/en/master/community_resources/package_structure/) requirements.
```suggestion
            "Ibexa\\Tests\\AdminUi\\": "tests/lib"
```

---

Fixed in ae4faf3e.

Includes added tests directory to code style configuration & phpunit.xml adjustment.